test/search-params.test.ts

```diff
@@ -241,4 +252,34 @@ describe('SearchParams', () => {
     const expected = 'user_query=title%3Afoo&debugging=true';
     expect(queryAsString).to.equal(expected);
   });
+
+  it('properly generates a URLSearchParam with a uid', async () => {
+    const query = 'title:foo';
+    const params = {
+      query,
+      uid: 'foobar',
+      includeClientUrl: false,
+    };
+    const urlSearchParam = SearchParamURLGenerator.generateURLSearchParams(
+      params
+    );
+    const queryAsString = urlSearchParam.toString();
+    const expected = 'user_query=title%3Afoo&uid=foobar';
+    expect(queryAsString).to.equal(expected);
+  });
+
+  it('properly generates a URLSearchParam with the client url by default', async () => {
+    const query = 'title:foo';
+    const params = {
+      query,
+    };
+    const urlSearchParam = SearchParamURLGenerator.generateURLSearchParams(
+      params
+    );
+    const queryAsString = urlSearchParam.toString();
+    const expectedPrefix = 'user_query=title%3Afoo&client_url=http';
```

Comment:
```suggestion
    const expectedPrefix = 'user_query=title%3Afoo&client_url=https';
```
can we expect `https`? 🙂

Reply:
Hmm, possibly? For the tests this will generally be a localhost URL -- not sure if we can always expect https everywhere the tests run. (At any rate, this test is really just verifying that the `client_url` param was added successfully)